docs/source/conf.py

```diff
@@ -19,8 +19,14 @@
 extensions = [
     "myst_parser",
     "sphinx_copybutton",
+    'sphinx_favicon',
```

Comment:
I think this is correct! And the only thing missing is adding `"sphinx-favicon"` to `pyproject.toml` as was done in https://github.com/kedro-org/kedro/pull/3547/files

Reply:
ah great, thanks for the suggestion, let me try that